src/main/java/com/github/manolo8/darkbot/config/actions/conditions/TargetTypeCondition.java

```diff
@@ -0,0 +1,89 @@
+package com.github.manolo8.darkbot.config.actions.conditions;
+
+import java.util.Locale;
+
+import org.jetbrains.annotations.NotNull;
+
+import com.github.manolo8.darkbot.Main;
+import com.github.manolo8.darkbot.config.actions.Condition;
+import com.github.manolo8.darkbot.config.actions.Parser;
+import com.github.manolo8.darkbot.config.actions.SyntaxException;
+import com.github.manolo8.darkbot.config.actions.ValueData;
+import com.github.manolo8.darkbot.config.actions.parser.ParseUtil;
+
+import eu.darkbot.api.game.entities.Npc;
+import eu.darkbot.api.game.other.Lockable;
+import eu.darkbot.api.game.other.EntityInfo.Diplomacy;
+
+@ValueData(name = "target-is", description = "Checks the target type", example = "target-is(Enemy)")
```

Comment:
for flexibility purposes, i believe this function should be getting for a specific ship as parameter, maybe similar to has-effect: `is-relationship(target(), NPC)` or similar